docs/en/latest/installation-guide.md

```diff
@@ -216,6 +216,22 @@ brew services start etcd
 
 ### Configuring APISIX
 
+:::note
+
+If you have done the Docker based installation, you need to exec into the `apisix` docker container to run the `apisix` command:
+
+```shell
+docker exec -it <name-of-container/container-id> sh
+```
+
+If you have installed via helm, you need to exec into the `apisix` kubernetes pod to run the `apisix` command:
+
+```shell
+kubectl exec -it <name-of-apisix-pod> -n <namespace-running-apisix> -- sh
+```
+
+:::
```

Comment:
Last ask, could you move this to after point "2"? It makes more sense to mention this after we mention "running the apisix command."